src/define_plugin_sets.h

```diff
@@ -447,6 +450,7 @@ To create/register a plugin, you have to :
     #define PLUGIN_SET_ONLY_SWITCH
     #define CONTROLLER_SET_STABLE
     #define NOTIFIER_SET_STABLE
+    #define USES_P029   // Output
```

Comment:
I doubt this one is needed, as it will be included when Domoticz is set.
If it isn't, then maybe the part where `USES_DOMOTICZ` is handled (line 233 - 239) should be placed after `USED_DOMOTICZ` is set.

Reply:
Well, ``USES_DOMOTICZ`` is only set by the minimal_IR sets, and then afterward, when any of the 3 Domoticz related controllers/plugins is set. So to make sure P029 is available for any Domoticz build I _also_ set it in the ``USES_DOMOTICZ`` logic. Kind of a chicken/egg dilemma...